release-notes.txt

```diff
@@ -1,4 +1,9 @@
----- Thursday 27th March, 2020 ----
+---- Friday 27th March, 2020 ----
+* Added Screen for PPE Removal
+* Added Screen for putting on PPE
+* Branded Home Screen
```

Comment:
No biggie, but you could also put in a point for the screens for taking off the PPE as thats what your working on here :-)